halfshell/source_s3.go

```diff
@@ -69,7 +69,9 @@ func (s *S3ImageSource) GetImage(request *ImageSourceOptions) (*Image, error) {
 }
 
 func (s *S3ImageSource) signedHTTPRequestForRequest(request *ImageSourceOptions) *http.Request {
-	imageURLPathComponents := strings.Split(request.Path, "/")
+	path := s.Config.PathPrefix + request.Path
```

Comment:
Building off of @rafikk’s [comment][1], what if this were just:
```go
path := s.Config.Directory + request.Path
```
since that exists, is unused, and feels like the same semantics?

[1]: https://github.com/oysterbooks/halfshell/pull/24#issuecomment-93807429